food-truck-frontend/src/pages/recommended-trucks.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import CreateTruckForm from "../components/CreateTruckForm";
+import CoolLayout from '../components/CoolLayout';
+import RecommendedTrucksForm from '../components/recommended-trucks/SearchRecommendedTrucksForm';
+
+function CreateTruckPage() {
+  return (
+    <CoolLayout>
```

Comment:
Would want to remove the layout from this, because we have the layout on every page by default.

Reply:
Yeah. That one's on me.